extensions/parquet/table/plugin/channelprovider/s3/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+dependencies {
+    api project(':extensions-parquet-base')
+
+    implementation project(':Base')
+    implementation project(':Util')
+    implementation project(':extensions-parquet-table')
```

Comment:
I think this dependency is just to support pulling in S3Instructions? I don't think we should need this dependency once S3Instructions moves?